client/my-sites/upgrades/domain-management/edit/card/header/primary-domain-button.jsx

```diff
@@ -30,6 +30,10 @@ const PrimaryDomainButton = React.createClass( {
 	},
 
 	render() {
+		if ( this.props.selectedSite && this.props.selectedSite.jetpack ) {
```

Comment:
I feel like this check should be moved to the parent component - makes it clearer that the button will not be rendered when we're dealing with a Jetpack site, IMHO. I we were conditionally rendering part of the component, that's cool (like it is in `domain-management/list` below). But here, we're just returning an empty component. I don't know - if I were digging into components, I'd prefer if this condition was not "hidden" in the render method, but in the parent component.